Documentation/ServiceUtilities/MixedRealityServiceRegistryAndIMixedRealityServiceRegistrar.md

```diff
@@ -0,0 +1,54 @@
+# What are the MixedRealityServiceRegistry and IMixedRealityServiceRegistrar
+
+The Mixed Reality Toolkit has two very similarly named components that perform related tasks: 
+MixedRealityServiceRegistry and IMixedRealityServiceRegistrar. What are they and what purpose does each serve?
```

Comment:
use [`` MixedRealityServiceRegistry`](xref:blah blah)? and for registrar

Reply:
added